config/settings.yml

```diff
@@ -141,3 +141,13 @@ paperclip:
   storage: filesystem
   url: ":rails_relative_url_root/:attachment/:id_partition/:style/:safe_filename"
   path: ":rails_root/public/:attachment/:id_partition/:style/:safe_filename"
+
+# saml:
+#   idp_metadata: "https://websso.example.com/idp/metadata"
+#   # certificate_file: path/to/file.p12 # Optional. Do not check in to version control.
```

Comment:
Is this the kind of thing we want to pick a path for and add to gitignore? Assuming we decide to use it--possibly this question is premature.

Reply:
I'm not quite sure where it should go if we end up using it. It's definitely something that should not be checked into version control.